### ISSUE TITLE
Add option to enable or disable bot respawn

### ISSUE THREAD

**Issue by Cyberschorsch** [splewis/csgo-practice-mode#226]
It would be handy if there would be an option to enable or disable the respawn behaviour of bots.

This could be an option when setting up the plugin.

**Comment by splewis** [comment]
Good idea, though I'll point out there is already a `sm_practicemode_bot_respawn_time` cvar, you could set it to a really high value (999999999) to effectively disable respawns.
